runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/DataflowRunner.java

```diff
@@ -2332,7 +2326,9 @@ static String getContainerImageForJob(DataflowPipelineOptions options) {
   }
 
   static boolean useUnifiedWorker(DataflowPipelineOptions options) {
-    return hasExperiment(options, "use_runner_v2") || hasExperiment(options, "use_unified_worker");
+    return hasExperiment(options, "beam_fn_api")
+        || hasExperiment(options, "use_runner_v2")
```

Comment:
IIRC, if we only specify `beam_fn_api`, we will go with JRH unless we have added runner v2 variants automatically somewhere.

Reply:
Good point. So now I check for any of the experiments and set them all if any of them are set.

Reply:
Thanks! That looks good.